metals/src/main/scala/scala/meta/internal/metals/debug/DebugProxy.scala

```diff
@@ -192,11 +204,15 @@ private[debug] final class DebugProxy(
       output.setOutput(msgWithoutColorCodes)
       message.setParams(output)
       client.consume(message)
-    case message => client.consume(message)
+
+    case message =>
```

Comment:
Why do we need to handle it in both server and client messages actually?

Reply:
Ah good call. We don't need the catch all for the client messages. The idea behind it would be

`client -> server initialize` starts the process and then _any_ message back from the server should stop it since in theory the first one coming back is either the initialize or something blowing up. Then a couple extra ones in case the client side tries to restart in the middle or if it's cancelled. I remove the one from the client.